### PR TITLE
Removed Snackbar

### LINKED ISSUE #12940: Add an `Enable` button to the `You must enable gestures to hide the answer buttons` snackbar
**Is your feature request related to a problem? Please describe.**
I'd like to ease the user life by not making them search the gestures preference with a simple `Enable` button

**Describe the solution you'd like**
1. Add an `Enable` button to the `You must enable gestures to hide the answer buttons` snackbar
2. If the user taps the button:
    - Open the controls fragment
    - Scroll/Highlight the `Enable gestures` preference
    - Turn on the `Enable gestures` preference

**Additional context**
IIRC, Android has a method to scroll to a preference on a PreferenceScreen.

And the SearchPreference library that we use has a highlight method that can serve as inspiration

### LINKED ISSUE #12995: The FullscreenMode must be on default value if gesture is disabled
###### Reproduction Steps

1. Enable Gestures
2. Select `Hide the system bars and answer buttons` (in FullscreenMode)
3. Disable Gestures 

###### Expected Result

The FullScreenMode  should be on `Off`

###### Actual Result

Here the FullScreenMode is still selected on `Hide the system bars and answer buttons`

###### Debug info

AnkiDroid Version = 2.16alpha91

Android Version = 10

Manufacturer = Google

Model = Android SDK built for x86

Hardware = ranchu

Webview User Agent = Mozilla/5.0 (Linux; Android 10; Android SDK built for x86 Build/QSR1.210802.001; wv) AppleWebKit/537.36 (KHTML, like Gecko) Version/4.0 Chrome/74.0.3729.185 Mobile Safari/537.36

ACRA UUID = ba1aa3b4-a9fe-4414-8ce5-fbb9b01159e1

New schema = false

Scheduler = std2

Crash Reports Enabled = false

DatabaseV2 Enabled = true

**For Better Understanding**

https://user-images.githubusercontent.com/65113071/208245765-6ba9ddbf-3334-444a-80f0-a0a0004494b3.mp4

###### Research
*Enter an [x] character to confirm the points below:*

- [x] I have read the [support page](https://ankidroid.org/docs/help.html) and am reporting a bug or enhancement request specific to AnkiDroid
- [x] I have checked the [manual](https://ankidroid.org/docs/manual.html) and the [FAQ](https://github.com/ankidroid/Anki-Android/wiki/FAQ) and could not find a solution to my issue
- [x] I have searched for similar existing issues here and on the user forum
- [x] (Optional) I have confirmed the issue is not resolved in the latest alpha release ([instructions](https://docs.ankidroid.org/manual.html#betaTesting))

### LINKED ISSUE #13002: Discuss if the "You must enable gestures first" snackbar should be shown if the answer buttons are hidden
Not that I'm thinking about it, even without the answer buttons and gestures, the user can still review their cards with a controller/keyboard/external device, so the snackbar isn't that useful and may not be even correct in its use.

_Originally posted by @BrayanDSO in https://github.com/ankidroid/Anki-Android/issues/12995#issuecomment-1356331577_

### PR DESCRIPTION
**Pull request by SanjaySargam**
## Purpose / Description
Showing snackbar isn't useful because we can even review card using controller/external keyboard

## Fixes
Closes #12986
Fixes #12940
Closes #12995 
Fixes #13002 

## How Has This Been Tested?

Virtual Emulator ( Pixel 6 Pro API 29 )

## Checklist
_Please, go through these checks before submitting the PR._

- [x] You have a descriptive commit message with a short title (first line, max 50 chars).
- [x] You have commented your code, particularly in hard-to-understand areas
- [x] You have performed a self-review of your own code
- [ ] UI changes: include screenshots of all affected screens (in particular showing any new or changed strings)
- [ ] UI Changes: You have tested your change using the [Google Accessibility Scanner](https://play.google.com/store/apps/details?id=com.google.android.apps.accessibility.auditor)

### CODE REVIEW

File: AnkiDroid/src/main/java/com/ichi2/anki/preferences/AppearanceSettingsFragment.kt [hunk 1]
```diff
@@ -46,27 +44,11 @@ class AppearanceSettingsFragment : SettingsFragment() {
...
-            if (prefs.getBoolean(GestureProcessor.PREF_KEY, false) || FullScreenMode.FULLSCREEN_ALL_GONE.getPreferenceValue() != newValue) {
-                true
-            } else {
-                // TODO add a button on the snackbar that leads directly to the
-                // Controls fragment and highlight the gesture preference
-                showSnackbar(R.string.full_screen_error_gestures)
-                false
-            }
+        requirePreference<ListPreference>(FullScreenMode.PREF_KEY).setOnPreferenceChangeListener { _, _: Any ->
+            true
```
Comment (BrayanDSO) [inline AnkiDroid/src/main/java/com/ichi2/anki/preferences/AppearanceSettingsFragment.kt:51]: Should be removed

**Review by BrayanDSO — CHANGES_REQUESTED** [review]

File: AnkiDroid/src/main/java/com/ichi2/anki/preferences/AppearanceSettingsFragment.kt (hunk 1 above)
Comment (SanjaySargam) [reply AnkiDroid/src/main/java/com/ichi2/anki/preferences/AppearanceSettingsFragment.kt:51]: Okk

**Review by BrayanDSO — APPROVED** [review]
LGTM. Something I suggest doing in the future is using `git amend` and `git rebase` to keep the commits organized and easier to review

**Comment by SanjaySargam** [comment]
> LGTM. Something I suggest doing in the future is using `git amend` and `git rebase` to keep the commits organized and easier to review

Ok 
Will take care next time

**Comment by criticalAY** [comment]
This closes #12986 and fixes #12940

**Review by criticalAY — APPROVED** [review]
LGTM

**Comment by criticalAY** [comment]
@BrayanDSO please remember to take care of the issue and the PR I mentioned above

**Comment by BrayanDSO** [comment]
> please remember to take care of the issue and the PR I mentioned above

Thanks. Edited the PR's text

**Comment by SanjaySargam** [comment]
@BrayanDSO 
This also fixes #12995

**Review by mikehardy — APPROVED** [review]
LGTM - confirmed (via direct inspection of crowdin activity timeline) strings activity quiet enough to just merge + resync after

**Pull request merged**